src/main/kotlin/racing/domain/Car.kt

```diff
@@ -0,0 +1,37 @@
+package racing.domain
+
+import racing.domain.strategy.MoveStrategy
+
+class Car(private var id: Int, private var distance: Distance = Distance.ZERO) : Movable, Comparable<Car> {
+
+    val getDistance: Distance
+        get() = distance
+
+    val getId: Int
+        get() = id
```

Comment:
코틀린은 프로퍼티의 getter/setter에 대한 접근자를 지원합니다. 이를 활용하면 다음과 같이 표현할 수 도 있을 것 같아요!
```suggestion
class Car(id: Int, distance: Distance = Distance.ZERO) : Movable, Comparable<Car> {

    var distance: Distance = distance
        private set

      var id: Int = id // id는 한 번 할당되면 변경되지 않을 텐데, `var`로 설정하신 의도가 있을까요???
        private set
```

Reply:
최초 기획의도는 레이싱 경주 중간중간 레코드를 저장하는 Repository가 존재하고, 그에 따른 CarRepository가 같이 존재했습니다. 그럼 Car엔티티의 아이디값은 실제로 DB에 저장되기전까지는 null이다가 저장 이후 id가 셋팅되기 때문에 val이 아닌 var로 지정했습니다. 

지금은 오버엔지니어링으로 판단되어 제거했지만, 차후 저장 요구가 있을껄 고려해서 var로 지정했습니다 👀